README.md

```diff
@@ -66,7 +66,7 @@ type BabelPlugin = string | [string, any] | [any, any];
 interface EmberCLIBabelConfig {
   /**
     Configuration options for babel-preset-env.
-    See https://github.com/babel/babel-preset-env#options for details on these options.
+    See https://github.com/babel/babel/tree/master/experimental/babel-preset-env#options for details on these options.
```

Comment:
If we land this as is, the link will be dead once `babel-preset-env` is moved out of the `experimental` subfolder. 

Can you update this link to instead point at:

https://github.com/babel/babel-preset-env/tree/v1.6.1#options

?

Reply:
Yeah sure!